Client/Frontend/Home/Wallpapers/v1/Interface/WallpaperManager.swift

```diff
@@ -59,8 +59,14 @@ class WallpaperManager: WallpaperManagerInterface, FeatureFlaggable, Loggable {
     }
 
     /// Determines whether the wallpaper onboarding can be shown
-    var canOnboardingBeShown: Bool {
-        guard featureAvailable,
+    func canOnboardingBeShown(using profile: Profile) -> Bool {
+        let cfrHintUtility = ContextualHintEligibilityUtility(with: profile)
+        let toolbarCFRShown = !cfrHintUtility.canPresent(.toolbarLocation)
```

Comment:
I think we will have the same issue that this [PR](https://github.com/mozilla-mobile/firefox-ios/pull/12076) is trying to fix. Because Toolbar CFR is setup to not show on iPad hence will prevent to show the onboarding on iPad